internal/venice-common/src/main/java/com/linkedin/venice/grpc/VeniceGrpcServer.java

```diff
@@ -74,27 +71,45 @@ private void initServerCredentials() {
   public void start() throws VeniceException {
     try {
       server.start();
+      LOGGER.info(
+          "Started gRPC server for service: {} on port: {} isSecure: {}",
+          config.getService().getClass().getSimpleName(),
+          port,
+          isSecure());
     } catch (IOException exception) {
       LOGGER.error(
-          "Failed to start gRPC Server for service {} on port {}",
+          "Failed to start gRPC server for service: {} on port: {}",
           config.getService().getClass().getSimpleName(),
           port,
           exception);
       throw new VeniceException("Unable to start gRPC server", exception);
     }
   }
 
-  public boolean isShutdown() {
-    return server.isShutdown();
+  public boolean isRunning() {
+    return !server.isShutdown();
   }
 
   public boolean isTerminated() {
     return server.isTerminated();
   }
 
+  private boolean isSecure() {
+    return !(credentials instanceof InsecureServerCredentials);
```

Comment:
Note that `null instanceof Anything` is always false. Right now, it doesn't seem like `credentials` can be null, but if they could be (e.g., due to some regression), then `isSecure` would return true (which would probably just result in a NPE sometime later, but in any event, it could be confusing?).

IDK what types can `credentials` take so IDK if a positive `instanceof` check is possible, or else maybe having a dedicated boolean flag for this, set in the same code path which sets the `credentials` prop. Or we can just leave it like this if you think it's fine. I don't have a strong opinion either way.

Reply:
Here `credentials` can not be null as always set it to some value in the constructor of `VeniceGrpcServer`.  But I agree this could be confusing hence I'v updated the code to use dedicated boolean flag.